src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal.js

```diff
@@ -377,7 +377,7 @@ export const CloudEnvironmentModal = ({
         ]),
         // Cloud environment button
         div({ style: toolButtonDivStyles }, [
-          isSettingsSupported(toolLabel, cloudProvider, workspace.accessLevel, workspace.workspace.createdDate) && h(Clickable, {
+          doesWorkspaceSupportCromwellAppForUser(workspace.workspace, cloudProvider, toolLabel) && h(Clickable, {
```

Comment:
hmm... is this correct? I believe we still want to check the original `isSettingsSupported`, which should defer to `doesWorkspaceSupportCromwellAppForUser` if the tool is Cromwell.

Reply:
I removed `isSettingsSupported` because it [only had check for Azure Cromwell app](https://github.com/DataBiosphere/terra-ui/blob/58dbf33e2f9baa134530d9fd2fa3d573ed8ee5c9/src/pages/workspaces/workspace/analysis/utils/tool-utils.ts#L179-L182). For all others it was simply returning true. Hence I removed it and instead here directly call `doesWorkspaceSupportCromwellAppForUser`. If you have strong opinion to keep the check for Settings button separate I can revert that change and have it call `doesWorkspaceSupportCromwellAppForUser` if the app is Cromwell.

Reply:
The reason it is named specifically after Cromwell is because it is being used in ContextBar and CloudEnvironment modal to also check if Cromwell icon and `Open` button can be displayed respectively. If you prefer I can rename it to `doesWorkspaceSupportAppForUser` (remove Cromwell from name) to make it more generic?